src/modules/billing/mappers/transaction.js

```diff
@@ -40,23 +41,38 @@ const mapDBToAgreements = row => {
   return agreements.filter(identity);
 };
 
+const doesVolumeMatchTransaction = (volume, transaction) => {
+  const transactionFinacialYear = transaction.endDate.slice(0, 4);
```

Comment:
I think this needs to consider the case where the charge period is not the full financial year. 
I think there is a helper for this:
```
const helpers = require('@envage/water-abstraction-helpers');
const transactionFinancialYear = helpers.charging.getFinancialYear(transaction.endDate);
```